plugins/modules/packaging/os/homebrew.py

```diff
@@ -127,6 +127,11 @@
     state: present
     install_options: with-baz,enable-debug
 
+- name: Install formula foo with 'brew' from cask
+  community.general.homebrew:
+    name: homebrew/cask/foo
+    state: present 
```

Comment:
```suggestion
    state: present
```

Reply:
@aminvakil - great catch! - just made a small push with that change, thank ya :+1: :)